Presentation.Web/app/components/it-system/usage/tabs/it-system-usage-tab-GDPR.view.html

```diff
@@ -38,6 +38,20 @@ <h4>{{systemUsageName}}</h4>
                            ng-blur="patch('DataProcessor', usage.dataProcessor)"
                            data-element-type="GDPRDataResponsible">
                 </div>
+                <!-- HostedAt -->
+                <div class="form-group">
+                    <label for="HostedAt" class="control-label small">IT systemet driftes</label>
+                    <select class="form-control input-sm" id="HostedAt"
+                            ui-select2="{minimumResultsForSearch: -1, allowClear: false}"
```

Comment:
Brug noSearchNoClearSelect2 fra scope. Du kan se den i controlleren. Det er for at vi ikke har kopieret denne select2 configuration alle steder